hack/update_submodules.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Update submodules HEAD to remote branches
+
+set -euo pipefail
+
+function help() {
+  echo "Usage: update_submodules.sh [OPTIONS] branch_name"
+  echo "Update submodules HEAD to remote branches"
+  echo "Must be executed in repo root directory"
+  echo ""
+  echo "Options:"
+  echo "-m   Specific modules to update, separated by spaces. Defaults to all submodules"
```

Comment:
specify*

Reply:
I dont really agree with that, what I put is grammatically correct and describes the value given to the arguments, as opposed to the action of using the flag.